README.md

```diff
@@ -7,7 +7,7 @@
 [![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
 [![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](http://www.firsttimersonly.com/)
 
-Welcome to freeCodeCamp's open source codebase and curriculum!
+Welcome to the freeCodeCamp's open source codebase and curriculum!
```

Comment:
This change is not needed.

Reply:
@raisedadead Is the commit approved or some more changes are needed?